s6/debian-root/etc/cont-init.d/20-start.sh

```diff
@@ -10,6 +10,10 @@ fi
 # used to start dnsmasq here for gravity to use...now that conflicts port 53
 
 $bashCmd /start.sh
+# Gotta go fast, no time for gravity
```

Comment:
Maybe not required, added before I added `--dns 1.1.1.1` which actually solved my gravity issues (known issue that never manifested in tests before)